src/components/about_component/About.jsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+
+import Lottie from "lottie-react";
+import aboutus_pc from "./aboutus_pc.json";
+import aboutus_coin from "./aboutus_coin.json";
+import styles from "./About.module.css";
+import styles2 from "./About2.module.css";
+
+function About() {
+  const [coinbox, setCoinbox] = useState(false);
+  const style = {
+    height: "16rem",
+  };
+  const style2 = {
+   
+    height: "25rem",
+    width: "25rem"
+  };
+  const handleEvent = () => {
+    setCoinbox(!coinbox);
+  };
+
+  // const interactivity = {
+  //   mode: "scroll",
+  //   actions: [
+  //     {
+  //       visibility: [0, 0.2],
+  //       type: "stop",
+  //       frames: [0],
+  //     },
+  //     {
+  //       visibility: [0.2, 0.45],
+  //       type: "seek",
+  //       frames: [0, 45],
+  //     },
+  //     {
+  //       visibility: [0.45, 1.0],
+  //       type: "loop",
+  //       frames: [45, 60],
+  //     },
+  //   ],
+  // };
+  //   <div>
+  //   <Lottie animationData={aboutus_pc} loop={true} autoplay={true} style={style}/>
+  //   <Lottie animationData={aboutus_coin}  loop={true} style={style}  />
+
+  //   </div  >
+  //   <div className="  w-[100%] bg-[url('./bg2.svg')]" >
+  //   <div className="flex w-[70.5625rem] h-[31.75rem] items-center gap-[7.6875rem] shrink-0 px-40 py-4">
+  // hello i am VIvek
+  //   </div>
+  //   </div>
```

Comment:
Remove the comments if not needed.

Reply:
The font name JetBrainsMono was not there in the public folder . That's why i added it

Reply:
Alright. Just define the font in App.css as did for other fonts and then use it wherever you need.